packages/hawtio/src/ui/page/HawtioPage.tsx

```diff
@@ -36,13 +39,12 @@ export const HawtioPage: React.FunctionComponent = () => {
   // navigate should be used in effect
   // otherwise "Cannot update a component (`BrowserRouter`) while rendering a different component" is thrown
   useEffect(() => {
-    if (!isLogin && !userLoading) {
+    if (!isLogin && userLoaded) {
```

Comment:
why this change? if user is loaded it should be already logged in. this change introduces a very different meaning to the behaviour here.
shouldn't we simply remove `!userLoading`?